### DISCUSSION TITLE
Allow configuration of AWS credentials via HostRules.

### DISCUSSION THREAD

**Discussion by Richard-de-Vries** [renovatebot/renovate#31030] [Suggest an Idea]
### Tell us more.

What would you like Renovate to be able to do?
I would like it if it was possible to configure the AWS credentials for the AMI and RDS data sources via the host rules.

Is this a feature you are interested in implementing yourself?
Sure :smile:

**Comment by rarkins** [comment]
Could you describe how this would look configuration-wise? Is it copy pasting similar support in other existing datasources?
